server/build.gradle

```diff
@@ -0,0 +1,32 @@
+
+// Apply the java plugin to add support for Java
+apply plugin: 'java'
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+    mavenCentral()
+}
+
+//Dependencies for the project
+dependencies {
+    compile 'org.slf4j:slf4j-api:1.7.12'
+    compile 'com.amazonaws:aws-lambda-java-core:1.1.0'
+    compile 'com.amazonaws:aws-lambda-java-events:1.1.0'
+    compile 'mysql:mysql-connector-java:6.0.2'
+    compile 'com.google.inject:guice:4.1.0'
+
+    testCompile 'org.testng:testng:6.9.10'
```

Comment:
You add TestNG dependency, but where are you telling gradle to run it during the test phase?

https://docs.gradle.org/current/dsl/org.gradle.api.tasks.testing.Test.html